src/components/CR8 - GalleryText/GalleryText.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+    <div class="gallery-text">
+        <!-- Title -->
+        <p class="title">Our Ecosystem</p>
+
+        <!-- Subtitle w/ Arrow -->
+        <p class="subtitle">
+            <span class="subtitle-text">We develop and produce original content for all genres and formats.</span>
+            <span class="prompt">
+                <p class="prompt-text">EXPLORE</p>
+                <span class="arrow">Arrow</span>
+            </span>
+        </p>
+
+        <!-- Block Links -->
+        <nuxt-link
+            v-for="(brand, index) in brands"
+            :key="index" 
+            class="brand-link"
+            @mouseover.native="set(brand.name)" 
+            @mouseout.native="set(null)"  
+            :to="`/${brand.name}`" tag="span"
+        >
+            <span class="logo">{{brand.name}} </span>{{brand.text}}
+        </nuxt-link>
```

Comment:
The slideshow should only occur when the logo is hovered, not the entire sentence about the brand.